src/legacy/legacy-default-xpring-client.ts

```diff
@@ -234,6 +234,21 @@ class LegacyDefaultXpringClient implements XpringClientDecorator {
       return feeAmount
     })
   }
+
+  public async accountExists(address: string): Promise<boolean> {
+    // TODO: replace with refactored checkNotXAddress() when complete
```

Comment:
Generally, prefer TODO's to have an owner so they don't get. 

So we could make this:
// TODO(amiecorso): Replace with refactored checkNotXAddress() when complete
// TODO(amiecorso): See asana.com/12345

Or we could drop it and just rely on the Asana task. 

All of these make me equally happy.

Reply:
i'll drop them because i've got the task!